template/report.html.tmpl

```diff
@@ -528,6 +528,12 @@
       <div id="requestEnv" class="hidden">{{ .FirstRequestEnv }}</div>
     </details>
 
+    <h3>LoadGen Command</h3>
+    {{ range .Data }}
+    <p>{{ .Name }}</p>
+    <pre>{{ with .TestResult.LoadGenOptions -}}{{ range $c := .Command }}{{ $c }} {{ end }}{{ end }}</pre>
```

Comment:
Notes about the weird templating language:

- The `-}}` serves to kill whitespace to the right, but here there's nothing to kill because the next instruction (`{{ range ... }}` follows immediately.

- `{{ range $c := .Command }}{{ $c }} {{ end }}` is equivalent to ` {{ range .Command }}{{ . }} {{ end }}`